smartapps/plaidsystems/spruce-scheduler.src/spruce-scheduler.groovy

```diff
@@ -812,8 +791,8 @@ private String getname(String i) {
 }
 
 private String zipString() {
-    if (!settings.zipcode) return "${location.zipCode}"
-    if (!settings.zipcode.isNumber()) return "pws:${settings.zipcode}"
+    if (!settings.zipcode && location.zipCode.isNumber()) return "${location.zipCode}"
+    if (!settings.zipcode) return "${location.latitude.floatValue()},${location.longitude.floatValue()}"//"pws:${settings.zipcode}"
```

Comment:
Same note as above re location fields.

Reply:
Check longitude and latitude are available before setting